extensions/applicationinsights-clickanalytics-js/src/events/PageAction.ts

```diff
@@ -126,6 +126,9 @@ export class PageAction extends WebEvent {
                 
                 pageActionEvent.timeToAction = _getTimeToClick();
                 pageActionEvent.refUri = isValueAssigned(overrideValues.refUri) ? overrideValues.refUri : _self._config.coreData.referrerUri;
+                if (_self._clickAnalyticsPlugin.core.config) {
+                    pageActionEvent.refUri = fieldRedaction(pageActionEvent.refUri, _self._clickAnalyticsPlugin.core.config);
```

Comment:
Do we want to redact URLs that are overridden and not automatically tracked?, in this case customer is explicitly adding this value right?

Reply:
I think it is a good point! Does the spec mention about any customer defined URLs? or should we add configs to allow disabling auto-redact urls?